src/azul/plugins/__init__.py

```diff
@@ -405,6 +408,18 @@ def aggregation_stage(self) -> 'Type[AggregationStage]':
     def filter_stage(self) -> 'Type[FilterStage]':
         raise NotImplementedError
 
+    @property
+    def version_format(self) -> str:
+        return '%Y-%m-%dT%H:%M:%S.%fZ'
```

Comment:
I still don't think this works for AnVIL. Please check with @noah-aviel-dove. 

More importantly, there may be additional places where this property would have to be used.

Reply:
This is almost identical to `dcp2_datetime_format` so I am worried about a) the brittle coupling and b) whether the difference is meaningful.